airflow/cli/__init__.py

```diff
@@ -15,3 +15,108 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+import os
+
+import rich_click as click
```

Comment:
Since this import is used only locally in this file, I’d prefer it use `rich_click`.

```suggestion
import rich_click
```

Reply:
That looks like a reasonable suggestion, but that import comes [straight out of the rich_click documentation](https://github.com/ewels/rich-click#import-as-click):

> ## Usage ##
> ### Import as click ###
> 
> To use rich-click, switch out your normal click import with rich-click, using the same namespace:
> 
> ```python
> import rich_click as click
> ```
> 
> That's it ✨ Then continue to use click as you would normally.

The rich_click package tries to be a superset of Click. When people are trying to add additional options or commands, I am trying to direct them to the Click documentation, since that is what's relevant to them.

If that justification doesn't sway you, I'm happy to change it if you insist (just confirm in a comment below).

Reply:
This got merged before I had a chance to change this.